src/strawman/collections/CollectionStrawMan6.scala

```diff
@@ -156,6 +159,8 @@ object CollectionStrawMan6 extends LowPriority {
     }
   }
 
+  type IndexedSeq[+A] = Seq[A] { def view: IndexedView[A] }
```

Comment:
If you do this then any call to `view` on a value with type `IndexedSeq[Foo]` will end up going through reflection, no?